Feat/button open logs directory (#250)
hotovo/aider-desk: CHANGELOG.md

- explicit cache control with Requesty Anthropic models used only when auto caching is disabled
- unified LLM providers settings
- improved onboarding flow with new unified LLM providers

> - button in aider about settings to show logs directory

Reviewer: ```suggestion
- added action to open directory with logs into Settings -> Version
```
